packages/ui-checkbox/src/Checkbox/index.tsx

```diff
@@ -138,66 +131,52 @@ class Checkbox extends Component<CheckboxProps> {
     }
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'e' implicitly has an 'any' type.
-  handleKeyDown = (e) => {
-    if (
-      this.props.variant === 'toggle' &&
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'return' does not exist on type 'CodesMap... Remove this comment to see the full error message
-      (e.keyCode === keycode.codes.enter || e.keyCode === keycode.codes.return)
-    ) {
-      // @ts-expect-error ts-migrate(2339) FIXME: Property '_input' does not exist on type 'Checkbox... Remove this comment to see the full error message
-      this._input.click()
+  handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (this.props.variant === 'toggle' && e.keyCode === keycode.codes.enter) {
```

Comment:
The previous version had:
```ts
if (e.keyCode === keycode.codes.enter || e.keyCode === keycode.codes.return) {...}
```
isn't this a regression?

Reply:
I think not. According to TS, return isn't found on keycodes.codes. Return and Enter is an alias for '13'.

https://github.com/timoxley/keycode/blob/c282ff62a4ff05a523d6026a66e83a4892e7f6e7/index.js#L77
https://github.com/timoxley/keycode/blob/c282ff62a4ff05a523d6026a66e83a4892e7f6e7/index.js#L134